app/controllers/BulkController.scala

```diff
@@ -55,27 +56,34 @@ class BulkController @Inject()(authAction: AuthAction,
 
   def getPreviousRequests(userId: String) = authAction.async {
     implicit request =>
+    {
+      Logger.debug(s"Request received: ${request}")
       repository.findByUserId(userId).map {
         case Some(x) => {
           Ok(Json.toJson(x))
         }
+        case _ => NotFound
       }
+    }
   }
 
   def getResultsSummary(userId: String, uploadReference: String) = authAction.async {
-    implicit request =>
+    implicit request => {
+      Logger.debug(s"Request received: ${request}")
```

Comment:
do we need to log this?

Reply:
In that case keep the loggers for time being just in case we need to investigate request info.